build.gradle

```diff
@@ -34,11 +35,12 @@ def versions = [
     reformHealthStarter: '0.0.4',
     reformLogging      : '5.1.0',
     reformS2sClient    : '3.0.0',
-    serenity           : '2.0.11',
+    serenity           : '2.0.23',
     sonarPitest        : '0.5',
     springBoot         : '2.1.4.RELEASE',
     springHystrix      : '2.1.1.RELEASE',
```

Comment:
is there a requirement to upgrde Serenity and Dependency check  for this PR?

Reply:
It was updated in master using one of the dependabot PR's while I was working on this branch. I'm not sure how dependabot works but the change was made on my PR too.